pkg/TIPA.py

```
@@ -519,23 +532,23 @@ def correct_text(self, wordlist: list) -> str:
         pattern = re.compile("|".join(rep.keys()))
         return pattern.sub(lambda m: rep[re.escape(m.group(0))], corrected_text).replace("__", "_").lstrip("()").strip("_-.,").replace("/", "_").replace("_Version", "")
-    
+
     def construct_search_url(self, site: str, search_text: str) -> str:
         if not search_text:
             return None
-        
+
         base_urls = {
             'market': 'https://www.google.com/search',
             'wiki': 'https://www.google.com/search'
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)